src/ga4gh/vrs/extras/translator.py

```diff
@@ -423,11 +431,10 @@ def _to_spdi(self, vo, namespace="refseq"):
         is expected to be normalized per VRS spec.
 
         """
-
-        if (type(vo).__name__ != "Allele"
-            or type(vo.location).__name__ != "SequenceLocation"
-            or type(vo.state).__name__ != "LiteralSequenceExpression"):
-            raise ValueError(f"_to_hgvs requires a VRS Allele with SequenceLocation and LiteralSequenceExpression")
+        if (vo.type != "Allele"
+            or vo.location.type != "SequenceLocation"
+            or vo.state.type != "LiteralSequenceExpression"):
+            raise ValueError("_to_hgvs requires a VRS Allele with SequenceLocation and LiteralSequenceExpression")
```

Comment:
It doesn't pop up until the _to_hgvs test, but looks like the issue is the output of _from_hgvs using deprecated classes

Reply:
^ I'll make an issue on VRS